moped-database/migrations/1654124226247_proj_list_view_project_soft_del/up.sql

```diff
@@ -0,0 +1,149 @@
+        ORDER BY mpn.date_created DESC
+        LIMIT 1) AS project_note,
+    ( -- get me the phase start of the most recently added construction phase entry
+      SELECT phases.phase_start
+      FROM moped_proj_phases phases
+      WHERE true
+        AND phases.project_id = mp.project_id 
+        AND phases.phase_name = 'construction'::text
+      ORDER BY phases.date_added DESC
+      LIMIT 1) AS construction_start_date,
+    ( -- get me the phase end of the most recently added completion phase entry
+      SELECT phases.phase_end
+      FROM moped_proj_phases phases
+      WHERE true 
+        AND phases.project_id = mp.project_id 
+        AND phases.phase_name = 'complete'::text
+      ORDER BY phases.date_added DESC
+      LIMIT 1) AS completion_end_date,
+    ( -- get me a list of the inspectors for this project
+      SELECT string_agg(concat(users.first_name, ' ', users.last_name), ', '::text) AS string_agg
+      FROM moped_proj_personnel personnel
+        JOIN moped_users users ON personnel.user_id = users.user_id
+        JOIN moped_project_roles roles ON personnel.role_id = roles.project_role_id
+      WHERE 1 = 1
+        AND roles.project_role_name = 'Inspector'::text
+        AND personnel.status_id = 1
+        AND personnel.project_id = mp.project_id
+      GROUP BY personnel.project_id) AS project_inspector,
+    ( -- get me a list of the designers for this project
+      SELECT string_agg(concat(users.first_name, ' ', users.last_name), ', '::text) AS string_agg
+      FROM moped_proj_personnel personnel
+        JOIN moped_users users ON personnel.user_id = users.user_id
+        JOIN moped_project_roles roles ON personnel.role_id = roles.project_role_id
+      WHERE 1 = 1
+        AND roles.project_role_name = 'Designer'::text
+        AND personnel.status_id = 1
+        AND personnel.project_id = mp.project_id
+      GROUP BY personnel.project_id) AS project_designer
+   FROM moped_project mp
+     LEFT JOIN project_person_list_lookup ppll ON mp.project_id = ppll.project_id
+     LEFT JOIN funding_sources_lookup fsl ON fsl.project_id = mp.project_id
+     LEFT JOIN project_type_lookup ptl ON ptl.project_id = mp.project_id
+     LEFT JOIN moped_entity me ON me.entity_id = mp.project_sponsor
+     LEFT JOIN moped_proj_partners mpp2 ON mp.project_id = mpp2.project_id AND mpp2.status_id = 1
+     LEFT JOIN moped_entity me2 ON mpp2.entity_id = me2.entity_id
+     LEFT JOIN LATERAL jsonb_array_elements(mp.task_order) task_order_filter(value) ON true
+  GROUP BY mp.project_uuid, 
+    mp.project_id, 
+    mp.project_name, 
+    mp.project_description, 
+    ppll.project_team_members, 
+    mp.project_description_public, 
+    mp.ecapris_subproject_id, 
+    mp.project_order, 
+    mp.current_status, 
+    mp.timeline_id, 
+    mp.current_phase, 
+    mp.end_date, 
+    mp.fiscal_year, 
+    mp.capitally_funded, 
+    mp.date_added, 
+    mp.added_by, 
+    mp.is_deleted, 
```

Comment:
I love how this PR identifies a problem that you just wouldn't know that the line above on 140 was changed since every migration is its own file .. **AND** then goes on to also include a solution to the problem. 💯 💯